### PR TITLE
Comments updates

### PR DESCRIPTION
**Pull request by shikhanansi**
@ephox-mogran I have updated this section wrt https://ephocks.atlassian.net/browse/DOC-253. Please review.

### CODE REVIEW

File: plugins/comments.md [hunk 1]
```diff
@@ -161,3 +184,14 @@ Follow this procedure to delete all conversations in the document:
...
 
 Check out the [Comments demo]({{site.baseurl}}/demo/comments-2) to try this new feature.
+
+## Using Comments 2.0 embedded mode with the Full Page plugin
+
+Users have to be cautious when deciding the order in which the plugins are added in the plugins list.
+
+Comments can cause an issue if the Full Page plugin `fullpage` appears before Comments 2.0 plugin `tinycomments` in the plugin list, and "tinycomments" is configured to use `embedded mode`.
+
+The consequence of this situation is that when a saved document is re-opened, the comment data is lost (but the highlights are still there). This is because the comment data is in the wrong place. The order that the plugins appear affects the order that the `getContent` hooks are processed in. The `fullpage` adds outer elements to the content and then `tinycomments` adds its comment data outside those markers. `tinycomments` expects to find its data inside the content, but having `fullpage` add outer `<html>` elements before `tinycomments` adds its comment data. This results in the comment data being lost
```
Comment (ephox-mogran) [inline plugins/comments.md]: adds outer*

File: plugins/comments.md [hunk 2]
```diff
@@ -14,6 +14,27 @@ Comments is a premium plugin from Tiny. Please see the [Premium features]({{site
...
 Once you have obtained the Comments 2.0 plugin, refer to the following instructions for using it.
 
+## Add Comments plugin
+
+Following is an example to add Comments 2.0 plugin to the TinyMCE editor:
+
+```js
+tinymce.init({
+  selector: '#tiny-ui .editor',
+  toolbar: 'bold italic underline | addcomment',
```
Comment (ephox-mogran) [inline plugins/comments.md]: I think TinyComments is in the default now (check in a fiddle), so you might be able to remove this.

File: plugins/comments.md [hunk 3]
```diff
@@ -94,6 +115,8 @@ tinymce.init({
 **Result**: The commented text will be highlighted yellow.
 ![**Highlighted text**]({{site.baseurl}}/images/highlight.png)
 
+> Note: The values provided in the above example are the default values provided in Comments 2.0. These can be changed as desired in the `tinymce.init` script.
```
Comment (ephox-mogran) [inline plugins/comments.md]: As discussed, this isn't quite right. Comments v1. still changes annotations via content_style, but Comments v2 has different classes, and it's best to use content skins.

File: plugins/comments.md (hunk 1 above)
Comment (ephox-mogran) [reply plugins/comments.md]: The last sentence is a bit confusing. Maybe change to 

```
The `fullpage` adds outer elements to the content and then `tinycomments` adds its comment data outside those markers, which puts them in the wrong place
```

File: plugins/comments.md [hunk 4]
```diff
@@ -54,20 +66,13 @@ tinymce.init({
 })
 ```
 
-Optional values: addcomment, showcomments
+**Optional values:** `addcomment`, `showcomments`
```
Comment (ephox-mogran) [inline plugins/comments.md]: I think this section is confusing. Maybe just consider deleting it (Optional Values and Result for the codepen).

File: plugins/comments.md [hunk 5]
```diff
@@ -54,20 +60,26 @@ tinymce.init({
...
+The `showcomments` toolbar button provides the ability to display comments field for the selected text. It is a toggle button and is used to hide the comments sidebar as well.
 
-* `addcomment` - Provides the ability to add comments.
-
-* `showcomments`- Provides the ability to display comments field for the selected text. It is a toggle button and is used to hide the comments sidebar as well.
+Use the following script to configure the Comments 2.0 toolbar button:
+```js
+tinymce.init({
+ selector: "#textarea",
+ toolbar: 'bold italic underline insertfile | showcomment',
```
Comment (ephox-mogran) [inline plugins/comments.md]: showcomments*

File: plugins/comments.md (hunk 1 above)
Comment (shikhanansi) [reply plugins/comments.md]: Fixed

File: plugins/comments.md (hunk 3 above)
Comment (shikhanansi) [reply plugins/comments.md]: Fixed

File: plugins/comments.md (hunk 4 above)
Comment (shikhanansi) [reply plugins/comments.md]: Fixed

File: plugins/comments.md (hunk 5 above)
Comment (shikhanansi) [reply plugins/comments.md]: Fixed

**Pull request merged**